aws/lambda/clickhouse-replicator-dynamo/lambda_function.py

```diff
@@ -0,0 +1,195 @@
+import json
+import os
+import re
+from collections import defaultdict
+from enum import Enum
+from typing import Any, Dict, Optional, Union
+from warnings import warn
+
+import clickhouse_connect
+
+
+DYNAMODB_TABLE_REGEX = re.compile(
+    "arn:aws:dynamodb:.*?:.*?:table/(?P<table>[0-9a-zA-Z_-]+)/.+"
+)
+CLICKHOUSE_ENDPOINT = os.getenv("CLICKHOUSE_ENDPOINT", "")
+CLICKHOUSE_USERNAME = os.getenv("CLICKHOUSE_USERNAME", "default")
+CLICKHOUSE_PASSWORD = os.getenv("CLICKHOUSE_PASSWORD", "")
+SUPPORTED_TABLES = {
```

Comment:
I wonder if we should take the opportunities to get rid of this arbitrary mapping and just use the same dynamo table names on ClickHouse, i.e. dynamo `torchci-workflow-job` goes into ClickHouse `torchci-workflow-job` table.  Same goes for S3.  The SQL queries will need to be updated, but that step is needed anyway.

Reply:
I'm leaving as is because I think CH requires using backticks for names with dashes, which I find really annoying